content/fr/service-discovery.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Découverte de services (Service Discovery)
+status: Completed
+category: Concept
+tags: ["réseau", "", ""]
+---
+
+La découverte de services est le processus permettant de découvrir les instances individuelles qui composent un service.
+Un outil de découverte de service garde une trace des différents nœuds ou points d'extrémité qui constituent un service.
+
+## Problème auquel il répond
+
+Les architectures cloud natives sont dynamiques et fluides, ce qui signifie qu'elles changent constamment.
+Une application [conteneurisée](/fr/containerization/) finira probablement par démarrer et s'arrêter plusieurs fois au cours de sa durée de vie.
+À chaque fois que cela se produit, elle aura une nouvelle adresse et
+toute application qui souhaite la trouver aura besoin d'un outil qui lui fournira les nouvelles informations de localisation.
+
+## Quel en est l'utilité
+
+Le mécanisme de découverte de services garde une trace des applications au sein du réseau afin qu'elles puissent se trouver mutuellement en cas de besoin.
+Il fournit une méthode courante pour trouver et potentiellement identifier les services individuels.
+Les moteurs de découverte de services sont des outils semblable à des bases de données qui stockent les informations sur les services existants et comment les localiser.
```

Comment:
Il manque un "s" sur "des outils semblables"

Reply:
fix in https://github.com/cncf/glossary/pull/2998/commits/6edac4e05b2075aace1c8a1166e3e80aacf06996